From 0eeccf24d41789ac40495e027ed16ff9e0c9657a Mon Sep 17 00:00:00 2001
From: Li Wang <li.wang@windriver.com>
Date: Fri, 5 Jul 2013 13:35:41 +0800
Subject: [PATCH 2/8] tty: compile warning: call function not match its
 declaration

warning: initialization from incompatible pointer type [enabled by default]

Signed-off-by: Li Wang <li.wang@windriver.com>
---
 drivers/tty/hvc/hvc_console.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/drivers/tty/hvc/hvc_console.c b/drivers/tty/hvc/hvc_console.c
index 8880adf..f18cb45 100644
--- a/drivers/tty/hvc/hvc_console.c
+++ b/drivers/tty/hvc/hvc_console.c
@@ -768,7 +768,8 @@ static int hvc_tiocmset(struct tty_struct *tty,
 }
 
 #ifdef CONFIG_CONSOLE_POLL
-int hvc_poll_init(struct tty_driver *driver, int line, char *options)
+int hvc_poll_init(struct tty_driver *driver, int line, char *options,
+		void *rx_callback)
 {
 	return 0;
 }
-- 
1.7.5.4

